tests/torch/test_compile_torch.py

```diff
@@ -1258,6 +1263,68 @@ def test_fancy_indexing_torch(model_object, default_configuration):
     compile_brevitas_qat_model(model, x, n_bits=4, configuration=default_configuration)
 
 
+@pytest.mark.parametrize(
+    "model_object",
+    [
+        pytest.param(MultiOutputModel),
+    ],
+)
+def test_multi_output(model_object, default_configuration):
+    """Test torch compilation with multi-output models."""
+    # Create model and random dataset
+    model = model_object()
+    x = numpy.random.randint(0, 2, size=(100, 3, 10)).astype(numpy.float64)
+    y = numpy.random.randint(0, 2, size=(100, 3, 10)).astype(numpy.float64)
+
+    # Pytorch baseline
+    torch_result = model(x[[0]], y[[0]])
+
+    # Compile with low bit width
+    quantized_module = compile_torch_model(
+        model, (x, y), n_bits=4, configuration=default_configuration
+    )
+    qm_result = quantized_module.forward(x[[0]], y[[0]])
+    simulation_result = quantized_module.forward(x[[0]], y[[0]], fhe="simulate")
+
+    # Assert that we have the expected number of outputs
+    assert isinstance(qm_result, tuple) and len(qm_result) == 2
+    assert isinstance(simulation_result, tuple) and len(simulation_result) == 2
+    assert isinstance(torch_result, tuple) and len(torch_result) == 2
+
+    # Assert that we are exact between simulation and clear quantized
+    for qm_res, sim_res in zip(qm_result, simulation_result):
+        assert isinstance(qm_res, numpy.ndarray)
+        assert isinstance(sim_res, numpy.ndarray)
+        assert array_allclose_and_same_shape(qm_res, sim_res, atol=1e-30)
+
+    # Assert that we aren't too far away from torch with low bit width
+    for qm_res, trch_res in zip(qm_result, torch_result):
+        assert isinstance(qm_res, numpy.ndarray)
+        assert isinstance(trch_res, numpy.ndarray)
+
+        # Very low tolerance because we use low-bit-width
+        assert array_allclose_and_same_shape(qm_res, trch_res, atol=1e-1)
```

Comment:
I think you mean `1e-10` right ?

Reply:
ah ok I was just surprised because you put the same comment (`# Very low tolerance because we use low-bit-width`) here and below (for  `atol=1e-10`, last check)

Reply:
Actually it's a typo in the second comment